packages/@react-spectrum/s2/src/Field.tsx

```diff
@@ -80,9 +81,10 @@ function FieldLabel(props: FieldLabelProps, ref: DOMRef<HTMLLabelElement>) {
         contain: {
           labelPosition: {
             top: 'inline-size'
-          }
+          },
```

Comment:
adding this allows the label be on a single line regardless of the picker's selected text